### ISSUE TITLE
N-HITS provided example in doc KeyError

### ISSUE THREAD

**Issue by Lynda-Starkus** [Nixtla/neuralforecast#459]
The "NHITS-median" column was renamed in newer version ? because it's returning a KeyError.

Was it renamed to "NHITS" ?

**Comment by kdgutier** [comment]
Hi @Lynda-Starkus,

Thanks for using the library. We have various examples where we use the `NHITS` models:
- https://nixtla.github.io/neuralforecast/examples/longhorizon_with_nhits.html
- https://nixtla.github.io/neuralforecast/examples/uncertaintyintervals.html

After fitting NHITS, depending on the loss function, one obtains different columns in the forecast data frame. By default, if one uses probabilistic losses, the output consists of quantiles and median; if one uses point forecast losses, the output is the model's name. Here is the losses catalog:
- https://nixtla.github.io/neuralforecast/losses.pytorch.html
